tests/stubs/pama-imaging-stub.js

```diff
@@ -0,0 +1,102 @@
+const uuid = require('uuid');
+
+function getReasons(reasons) {
```

Comment:
Nit: I might call this `makeReasons` given that it's creating something new rather than extracting data out of the supplied payload. (No need to change if you prefer it this way though; certainly I can see the case.

Reply:
Nice.  I made the change while I was in there adding some code coverage to the unit test.